src/subs/Match.cpp

```diff
@@ -186,18 +186,148 @@ Match::Match(unique_ptr<Detection> e)
           isSolved(false),
           trivialWeights(false) {
     add(std::move(e));
+    // Set null field projection.
+    astrometry::Orientation orient(astrometry::SphericalICRS(0, 0));
```

Comment:
It looks like you might be using this value because you think you need to populate `fieldProjection` with _something_ even though it shouldn't ever be used?  If that's the case, you might want to only put the new `fieldProjection` member in `PMMatch`, which will get you a compile error if any code in `Match` tries to use it.